packages/core-types/tests/integration/components/field-editors/integer-editor-test.js

```diff
@@ -40,17 +32,29 @@ module('Integration | Component | field editors/integer editor', function(hooks)
   });
 
   test('it can be disabled', async function(assert) {
-    this.setProperties({
-      model: EmberObject.create({ rating: 3 }),
-      onchange() {}
-    });
+    this.set('model', EmberObject.create({ rating: 3 }));
     await render(hbs`
       {{field-editors/integer-editor
         content=model
         field="rating"
-        onchange=onchange
         disabled=true
       }}`);
     assert.dom('input').isDisabled();
   });
+
+  test('onchange is called when the field is left', async function(assert) {
+    let changed;
+    this.set('onchange', () => {
+      changed = true;
+    });
+    this.set('model', EmberObject.create({ rating: 3 }));
+    await render(hbs`
+      {{field-editors/integer-editor
+        content=model
+        field="rating"
+        onchange=onchange
+      }}`);
+    await triggerEvent('input', 'blur');
```

Comment:
there is a dedicated [`blur()`](https://github.com/emberjs/ember-test-helpers/blob/master/API.md#blur) helper, which has a bit more code in it to make it work properly on all the browsers

Reply:
Didn't know about this but it'd be great as I've had problem with triggerEvent in the past. For some reason, `await blur('input')` doesn't work, though, it seems like it doesn't trigger the event.

Reply:
Ignore this, the selector (`input`) is not specific enough so it finds another one. Interesting that it has worked with `triggerEvent` though.

Reply:
There's still a timing issue in there. The test fails by default but if I put in a breakpoint in the `blur` method (I did this to check if the event is fired on the right element) it passes.

Reply:
Yep, that works!